src/EditorFeatures/CSharpTest/RefactoringHelpers/RefactoringHelpersTests.cs

```diff
@@ -514,6 +514,47 @@ void M()
         #endregion
 
         #region Attributes
+        [Fact]
+        [WorkItem(37584, "https://github.com/dotnet/roslyn/issues/37584")]
```

Comment:
nit: i just checked and the vast majority of tests do `[Fact, WorkItem...` when both attributes are in use.  Consider switching to that as it just keeps things tidier.  but up to you.